src/test/regress/expected/multi_utilities.out

```diff
@@ -404,21 +404,6 @@ SELECT pg_size_pretty( pg_total_relation_size('local_vacuum_table') );
  21 MB
 (1 row)
 
------------------ PROCESS_TOAST is only available for pg14
```

Comment:
why do we remove these tests?

Reply:
Process_toast tests are already in pg14.sql file. It should not be in multi_utilities.sql because auto option for index_cleanup is only valid for pg14.

Reply:
But these tests are on a local table and potentially testing the behavior on the local tables. I don't think we should remove these. In other words, I'm not convinced that we should remove if you don't have a strong opinion on this.

If you really want to remove these, it is better to open a separate PR and describe why you want to remove so that we can focus on those.

Reply:
Those tests are added by myself when I work on #5946. They never run before because they were commented out. When I remove comments, they fail for postgres build 13 because there is no support for process_toast on pg13. Thus, we need to run process toast tests only for postgres version 14. That is why I moved those process toast tests to pg14.sql file.

Reply:
My point is that pg14.sql file does NOT have as detailed tests as this one, so why not we move these tests there?

I'm fine if you share the exact tests that are on pgg14.sql that completely covers here. Otherwise, I'd suggest we move these tests to PG 14.sql file and uncomment them